data/json/mapgen/gas_stations/s_gas.json

```diff
@@ -49,7 +49,8 @@
       "place_liquids": [
         { "liquid": "gasoline", "x": 2, "y": 3, "repeat": [ 200, 1075 ] },
         { "liquid": "diesel", "x": 4, "y": 3, "repeat": [ 200, 1075 ] }
-      ]
+      ],
+      "place_monsters": [ { "monster": "GROUP_ZOMBIE_GAS", "x": [ 5, 10 ], "y": [ 6, 10 ], "chance": 2 } ]
```

Comment:
`place_monsters` calculates chance a bit differently. Here, `chance` represents a 1 in N chance that the entire group will spawn (according to the mapgen doc). This means gasoline zeds have a 50% chance of spawning.